### PR TITLE
[3.4.0] Docs skeleton component

### PR DESCRIPTION
**Pull request by svdimitr**

### CODE REVIEW

File: components/skeleton/overview.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+description: Overview of the Skeleton for Blazor.
+slug: skeleton-overview
+tags: telerik,blazor,skeleton,overview
+published: True
+position: 0
+---
+
+# Blazor Skeleton Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/skeleton" target="_blank">Blazor Skeleton</a> is a loading container mainly used for indicating to users that the page content is currently loading. What separates it from the conventional loaders is that it mimics the page layout by showing its elements in a shape similar to the actual content as it loads.
```
Comment (dimodi) [inline components/skeleton/overview.md:13]:
```suggestion
The <a href = "https://www.telerik.com/blazor-ui/skeleton" target="_blank">Blazor Skeleton</a> is a loading indicator. What separates it from conventional loaders is that it mimics the page layout by showing elements in a similar shape as the actual content that will appear after loading.
```

File: components/skeleton/overview.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+tags: telerik,blazor,skeleton,overview
+published: True
+position: 0
+---
+
+# Blazor Skeleton Overview
+
+The <a href = "https://www.telerik.com/blazor-ui/skeleton" target="_blank">Blazor Skeleton</a> is a loading container mainly used for indicating to users that the page content is currently loading. What separates it from the conventional loaders is that it mimics the page layout by showing its elements in a shape similar to the actual content as it loads. 
+
+
```
Comment (dimodi) [inline components/skeleton/overview.md:15]:
```suggestion
```

File: components/skeleton/overview.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+## Creating Skeleton
```
Comment (dimodi) [inline components/skeleton/overview.md:17]:
```suggestion
## Creating Blazor Skeleton
```

File: components/skeleton/overview.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+}
+
+@code {
+    string Text { get; set; } = string.Empty;
+    bool isVisible { get; set; }
+
+    protected override async Task OnInitializedAsync()
+    {
+        isVisible = true;
+        await Task.Delay(1000);
```
Comment (dimodi) [inline components/skeleton/overview.md:48]: One second may be too little to appreciate the animation.

```suggestion
        await Task.Delay(2000);
```

**Review by dimodi — APPROVED** [review]
Approving, with some suggestions. The important bits are -
* Consider two Skeletons in the Basic example;
* Explain the difference between `Text` and `Rectangle`;

File: components/skeleton/appearance.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+* [AnimationType](#animationtype)
+
+
+## ShapeType
+
+The `ShapeType` parameter sets the shape of the Skeleton. It takes a member of the `SkeletonShapeType` enum:
+
+* `Text` (default)
+* `Rectangle`
+* `Circle`
```
Comment (KristianMariyanov) [inline components/skeleton/appearance.md:27]: We can specify that the circle shape can be used for creating ellipses as well.

**Pull request merged**